test/test_topic.py

```diff
@@ -226,29 +219,27 @@ def callback(msg, nodedata):
 
         self.blackboard = Blackboard()
         self.topic_monitor = topic.TopicMonitor(name="monitor",
-                                                topic_name="/topic_monitor",
+                                                topic_name="topic_monitor",
                                                 topic_type=String,
                                                 cb=callback,
                                                 blackboard=self.blackboard)
 
         self.blackboard.save("count", 0, self.topic_monitor._id)
-        self.msg_pub = rospy.Publisher("/topic_monitor", String,
-                                       queue_size=1, latch=True)
+
+        assert_equal(rospy.Subscriber, type(self.topic_monitor.topic_sub))
```

Comment:
Whats this line checking?  Is it that the type is the same or that they are the same object?